test/lib/ufe/test-samples/parentCmd/simpleSceneMayaPlusUSD_TRS.ma

```diff
@@ -0,0 +1,242 @@
+//Maya ASCII 2020 scene
```

Comment:
A test scene that has a simple Maya scene, and the USD version of it under a proxy shape.  This makes comparison between Maya parenting and USD parenting behavior trivial: just perform the operation on the Maya objects, and the USD objects should behave the same and have the same transform values.